src/server/routes.rs

```diff
@@ -146,7 +147,8 @@ pub async fn search(
 /// Handles the route of robots.txt page of the `websurfx` meta search engine website.
 #[get("/robots.txt")]
 pub async fn robots_data(_req: HttpRequest) -> Result<HttpResponse, Box<dyn std::error::Error>> {
-    let page_content: String = read_to_string("./public/robots.txt")?;
+    let page_content: String =
+        read_to_string(format!("{}/robots.txt", handle_different_theme_path()?))?;
```

Comment:
I may seem confusing as to why the `robots.txt` file is located under the theme path. Since the theme path is essentially the public path, I think it could be better to consider a function name such as `handle_differenct_public_path` instead of `handle_different_theme_path`. What do you say? @neon-mmd

Reply:
My thinking was that the name `theme` would be more appropriate while writing the code but now I think you are right, I will make the change right away :slightly_smiling_face: .